DataFormats/Detectors/HMPID/include/DataFormatsHMP/Cluster.h

```diff
@@ -120,6 +120,16 @@ class Cluster
 };
 
 } // namespace hmpid
+
```

Comment:
please fix the Cluster class, it should not contain the pointer on the vector, instead the charges should be accessed from the separate vector.